configuration/src/core.rs

```diff
@@ -27,40 +27,62 @@ pub struct EvmCoreContracts {
     pub replicas: HashMap<String, Proxy>,
 }
 
+/// Empty Substrate contracts
+#[derive(Default, Copy, Debug, Clone, serde::Serialize, serde::Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct SubstrateCoreDeployInfo {
+    /// Contract Deploy Height
+    #[serde(default, deserialize_with = "deser_nomad_u32")]
+    pub deploy_height: u32,
+    // TODO: add replicas for substrate rollout v2
+}
+
 /// Core Contract abstract
 #[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
 #[serde(untagged)]
-pub enum CoreContracts {
+pub enum CoreDeploymentInfo {
     /// EVM Core
-    Evm(EvmCoreContracts),
-    // Substrate (empty)
+    Ethereum(EthereumCoreDeploymentInfo),
+    /// Substrate core
+    Substrate(SubstrateCoreDeployInfo),
 }
 
-impl CoreContracts {
+impl CoreDeploymentInfo {
     /// Get an iterator over the replicas present in this deploy
     pub fn replicas(&self) -> impl Iterator<Item = &String> {
         match self {
-            CoreContracts::Evm(contracts) => contracts.replicas.keys(),
+            CoreDeploymentInfo::Ethereum(contracts) => contracts.replicas.keys(),
```

Comment:
Nit: Consistent use of brackets for these

Reply:
`cargo fmt` does this automatically. Would fail lint if these were not consistent